### PR TITLE
feat(frontend): allow managing multiple nodes on one page

### PR DESCRIPTION
**Pull request by ravenclaw900**

### CODE REVIEW

**Comment by ravenclaw900** [comment]
Well, the viewing of different nodes works, but, for instance, the update message pushes the node selection box out of the way, which pushes the light/dark mode button out. How should we go about this? I'm thinking something like a "messages" icon, which will reveal the update and any other messages, and maybe a "settings" icon for the node selection box, and possibly themes once those get implemented.

File: config.toml [hunk 1]
```diff
@@ -12,4 +12,9 @@
 # To get hash: "echo -n '<PASSWORD>' | sha512sum"
 #hash = "SHA512 hash of password"
 # To get random secret: "openssl rand -hex 32" 
-#secret = "64-character secret"
\ No newline at end of file
+#secret = "64-character secret"
+# Other nodes viewable on frontend page
+# - Defualt: []
```
Comment (MichaIng) [inline config.toml:17]: Little type 🙂: `Defualt` => `Default`

**Comment by MichaIng** [comment]
A notification button/icon seems a good idea. In theory, it could be also shown conditionally, i.e. either if there are multiple notifications or when not everything fits into the viewport with a single notification expanded.

**Comment by ravenclaw900** [comment]
Alright, notification button added. If that looks good, I'm going to merge and release.

**Comment by MichaIng** [comment]
I'll do some testing later today, many thanks for implementing 👍.

**Pull request merged**